### PR TITLE
Implemented consumer builder

### PR DESCRIPTION
**Pull request by IdoRozval**

### CODE REVIEW

File: Blizzard.IO.RabbitMQ.Builders/NetqRabbitConsumerBuilder.cs [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        public NetqRabbitConsumerBuilder<TData> AddConverter(IConverter<MessageProperties, RabbitMessageProperties> converter)
+        {
+            _converter = converter;
+            return this;
+        }
+
+        public NetqRabbitConsumer<TData> Build()
+        {
+            IBus bus = InitConnection();
+            _sourceQueue = _sourceQueue ?? bus.DeclareQueue("DefaultQueue");
```
Comment (alloncm) [inline Blizzard.IO.RabbitMQ.Builders/NetqRabbitConsumerBuilder.cs]: Create another variable dont use the field again it is less readable like that

Comment (IdoRozval) [reply Blizzard.IO.RabbitMQ.Builders/NetqRabbitConsumerBuilder.cs]: `??` is a c# feature, why not to use it?

Comment (alloncm) [reply Blizzard.IO.RabbitMQ.Builders/NetqRabbitConsumerBuilder.cs]: use this feature but assign the result to a new varaible it is confusing and looks weird to assign the same variable to itself

Comment (IdoRozval) [reply Blizzard.IO.RabbitMQ.Builders/NetqRabbitConsumerBuilder.cs]: f

File: Blizzard.IO.RabbitMQ.Builders/NetqRabbitConsumerBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+        public NetqRabbitConsumerBuilder<TData> AddConverter(IConverter<MessageProperties, RabbitMessageProperties> converter)
+        {
+            _converter = converter;
+            return this;
+        }
+
+        public NetqRabbitConsumer<TData> Build()
+        {
+            IBus bus = InitConnection();
+            RabbitQueue sourceQueue = _sourceQueue ?? bus.DeclareQueue("DefaultQueue");
```
Comment (alloncm) [inline Blizzard.IO.RabbitMQ.Builders/NetqRabbitConsumerBuilder.cs]: put this string in a const

Comment (IdoRozval) [reply Blizzard.IO.RabbitMQ.Builders/NetqRabbitConsumerBuilder.cs]: f

**Pull request merged**